### PR TITLE
Update to Collections, NTP and tips

### PR DESCRIPTION
**Pull request by lilliankravitz**
Collections price comparisons
NTP pre render
Tips: in product recommendations

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper.md [hunk 1]
```diff
@@ -70,6 +69,8 @@ Every 24 hours Microsoft Edge downloads a list of supported sites for which spec
 
 To delete the templates stored on the device and clear the cache data, go to `edge://settings/privacy` and under **Clear browsing data** next to **Clear browsing data now** select the **Choose what to clear** button, select the desired time range and type of data, then select the **Clear now** button.  Another way to delete cached data, go to `edge://settings/clearBrowserData` select the desired time range and type of data, then select the **Clear now** button.  
 
+To help you get the best deal while shopping and using Collections, if you add a product to your Collections, Microsoft Edge will offer the ability to compare the product price to other retailers.  If you select Compare price to other retailers Microsoft Edge will send the product name, shopping domain, price and other product details about the product to Bing to compare prices to other retailers.  The product Collections info will be sent over HTTPS. These HTTPS calls do not contain any user identifiers.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper.md:72]:
```suggestion
To help you get the best deal while shopping and using Collections, if you add products to your Collections, Microsoft Edge will offer the ability to compare the product price to other retailers.  If you select Compare price to other retailers Microsoft Edge will send the product name, shopping domain, price and other product details about the product to Bing to compare prices to other retailers.  The product information in Collections will be sent over HTTPS. These HTTPS calls do not contain any user identifiers.  
```

File: microsoft-edge/privacy-whitepaper.md [hunk 2]
```diff
@@ -250,7 +251,9 @@ Microsoft Edge uses a Microsoft network time service to track time from an exter
...
+Microsoft Edge provides an engaging and user-centric new tab page experience with a search box powered by [Bing][BingMain], quick link tiles for the sites you visit most frequently, and relevant content from Microsoft News or Office 365.  Customize the appearance of the new tab page by selecting the customize button.  Your new tab page preferences are set for each profile and stored locally on your device, and the preferences are not synced across devices.
+
+To enhance the load times of the new tab page, the Microsoft new tab page may be loaded in the background to make it faster. The content that is loaded might include cookies, if you allow cookies. To disable loading the Microsoft new tab page in the background, go to `edge://settings/newTabPage` and disable the setting **Preload the new tab page for a faster experience**.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper.md:256]:
```suggestion
To enhance the load times of the new tab page, the Microsoft new tab page may be loaded in the background to make it faster. The content that is loaded might include cookies, if you allow cookies. To disable loading the Microsoft new tab page in the background, go to `edge://settings/newTabPage` and turn off the setting **Preload the new tab page for a faster experience**. 
```

File: microsoft-edge/privacy-whitepaper.md [hunk 3]
```diff
@@ -410,7 +413,7 @@ To help resolve URL typos in the address bar that result in a website error, Mic
 
 Signing into Microsoft Edge with a Microsoft account allows you to sync your browsing data across all your signed-in versions of Microsoft Edge.  You may sync your favorites, settings, form fill data \(addresses and more\), passwords, extensions, and collections.  You must allow permission to turn on sync in Microsoft Edge, and each synced data type may be turned on or off individually.  Favorites includes any tabs you previously set aside in the legacy version of Microsoft Edge, which sync along with the rest of your favorites.  Deleted or modified favorites or other data from one signed-in version of Microsoft Edge sync to all other signed-in versions of Microsoft Edge where sync is turned on.  To manage sync configurations, go to `edge://settings/profiles/sync`.  Your sync settings may be managed by your organization.
 
-:::image type="complex" source="./media/whitepaper-media/sync.png" alt-text="Sync" lightbox="./media/whitepaper-media/sync.png":::
+:::image type="complex" source="./media/whitepaper-media/sync.png" alt-text="image of sync being on" lightbox="./media/whitepaper-media/sync.png":::
    Sync  
 :::image-end:::
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper.md:418]:
```suggestion
:::image type="complex" source="./media/whitepaper-media/sync.png" alt-text="Image of sync setting being set to on" lightbox="./media/whitepaper-media/sync.png":::
   Sync setting is on
:::image-end:::  
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

**Comment by lilliankravitz** [comment]
#sign-off

**Review by Reezaali — APPROVED** [review]

**Pull request merged**